.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,23 +10,70 @@ assignees: ''
 
 The release number or commit hash of the version you're using.
 
-## Operating system & version
+## \[Optional\] Setup information
 
-Ex: Windows 10, Ubuntu 18.04, iOS 14.
+If you are reporting a bug about:
+
+- difficulties getting Bevy to build or run on your machine
+- unusual rendering bugs
+- unusual input bugs
+- hardware specific problems
+
+These issues are often specific to hardware or OS configuration and can be challenging to reproduce.
+As a result, please consider including information about:
+
+- the Rust version you're using
+  - Bevy has a policy of relying on the "latest stable release" of Rust
+  - Nightly should generally work, but there are sometimes regressions: please let us know!
+- the operating system, including its version
+  - e.g. Windows 10, Ubuntu 18.04, iOS 14.
+- relevant hardware
```

Comment:
Instead, I would recommend to paste the adapter info log that shows up when launching any example. It contains most of the relevant hardware info.

The on that looks like this:
```log
AdapterInfo { name: "NVIDIA GeForce RTX 2070", vendor: 4318, device: 7938, device_type: DiscreteGpu, backend: Vulkan }
```